### DISCUSSION TITLE
Is Flyway 10 support planned?

### DISCUSSION THREAD

**Discussion by b-thiswatch** [quarkusio/quarkus#39516] [Community]
Hi,

we are using Flyway in our project and would like to upgrade to Postgres 16.2 in the near future. The currently used Flyway 9.22 does not officially support PG 16 (it works, though). Are you planning to upgrade Flyway to V10?

Thanks

**Comment by gsmet** [comment]
Hi! We actually merged the upgrade during the week-end: https://github.com/quarkusio/quarkus/pull/39509

**Reply by gsmet** [reply]
It will be in Quarkus 3.10 for sure. I asked on the quarkus-dev ML what people would think of pushing the upgrade to 3.9 even if we are post-CR1. I will let you know how it goes.

**Comment by b-thiswatch** [comment]
Great!

Sorry, I searched the issues and zulip, but forgot to check the commits...

**Reply by gsmet** [reply]
Oh no worries, it was perfectly fine to ask :).
